MelonLoader/Melons/MelonMod.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Reflection;
+#pragma warning disable 0618
```

Comment:
Please add a comment stating which warning is being disabled and why

Reply:
Ight

Reply:
https://github.com/SlidyDev/MelonLoader/commit/f2c74f48e912f175c2d7b55d70e02d3579a57d46